testplan/testing/multitest/entries/base.py

```diff
@@ -272,9 +272,15 @@ class Graph(BaseEntry):
     """Create a graph for the report."""
     def __init__(self, graph_type, graph_data, description=None,
                  series_options=None, graph_options=None):
-        valid_graph_types = ['Line', 'Scatter', 'Bar', 'Whisker',
+        """
+        NOTE:
+        When adding functionality to Graph, VALID_GRAPH_TYPES,
+        VALID_CHART_TYPES, VALID_GRAPH_OPTIONS and
+        VALID_SERIES_OPTIONS must be kept updated
+        """
+        VALID_GRAPH_TYPES = ['Line', 'Scatter', 'Bar', 'Whisker',
```

Comment:
Nice, it's better to store conceptual constants as ALL_CAPS (python doesn't have real constants, so marking them clearly by convention is useful)

Only comment is that maybe these constants could be placed on the class itself (i.e. same indentation as the `def __init__(...):` and accessed through `self.VALID_CHART_TYPES` etc. That's quite a common pattern for constants that belong to a particular class.